README.rst

```diff
@@ -0,0 +1,99 @@
+Blue State Digital Python API Client
+====================================
+
+A Python SDK for the Blue State Digital API. Stay Progressive.
+
+.. image:: https://travis-ci.org/bmd/bsd-api-python.svg?branch=master
+    :target: https://travis-ci.org/bmd/bsd-api-python
+
+.. image:: https://coveralls.io/repos/github/bmd/bsd-api-python/badge.svg
+    :target: https://coveralls.io/github/bmd/bsd-api-python
+
+
+Requirements
+============
+
+* Python 2.7+
+* Python 3.3+
+
+Installing
+==========
+
+Install via PyPI.
+
+.. code-block:: bash
+
+    $ pip install bluestatedigital-api
+
+Using the library
+=================
+
+Configuring Your API User
+-------------------------
+
+Before you can use the Tools API Library, you'll need to configure an API user in the Tools. You can edit an existing user or create a new one from the "Administrator Tools > Manage API Users" page. The ``api_id`` and ``secret`` are taken directly from the Manage API Users page.
+
+API Users in the Tools are granted granular permissions, and can be configured to access - or be prevented from accessing - each API endpoint. You can find a complete list of Tools API endpoints `here <http://tools.bluestatedigital.com/page/api/doc>`_.
+
+
+Using the library
+-----------------
+
+Basic usage
+~~~~~~~~~~~
+
+.. code-block:: python
+
+    from bsdapi import BsdApiClient
+
+    # instantiate the API Client
+    client = new BsdApiClient(
+        'MY_API_ID',
+        'MY_API_SECRET',
+        'https://my.tools.url.net/'
+    )
```

Comment:
The order of the args and object creation step is incorrect in the docs here

It should be

```python
client = new BsdApiClient(
    'https://my.tools.url.net/',
    'MY_API_ID',
    'MY_API_SECRET'
)
```